src/jlgen.jl

```diff
@@ -226,12 +226,17 @@ end
 
 @static if isdefined(Base.Experimental, Symbol("@overlay"))
 using Core.Compiler: OverlayMethodTable
+@static if VERSION ≥ v"1.9.0-DEV.120"
```

Comment:
`@static` isn't needed for toplevel code AFAIK

Reply:
Also, just use `>=`, people can use fonts with ligatures if they want compact notation :-)

Reply:
> `@static` isn't needed for toplevel code AFAIK

I think it's still useful to annotate them, mostly because their existence indicates there is some version-specific code and so a grep with `@static` might be useful when we want to remove code that comes to be staled in the future.

Having said that, I'm totally okay with remove them if you don't like ;)

Reply:
> Having said that, I'm totally okay with remove them if you don't like ;)

Yeah I understand the motivation, but I also appreciate if the style of doing things is a little consistent across the codebase :-) Same with comments on `else` branches, we don't do that anywhere here (and it also doesn't seem very useful for such short blocks).

Reply:
ok, removed!

Reply:
Ah, they were required for not expanding `@MethodTable` on 1.6...

Reply:
agh...